### PR TITLE
haskell: fix removed aliases; fix OfBorg eval

### PR DESCRIPTION
**Pull request by ryantm**
###### Motivation for this change

Fix master eval. Looks like `pkgs.gnome3.gtk` doesn't exist anymore, so we need to fix this hackage-packages update.

cc @peti @jtojnar 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
I would just fix it manually in before we update the `cabal2nix`.

**Comment by jtojnar** [comment]
See https://github.com/NixOS/cabal2nix/pull/410

**Comment by peti** [comment]
Yes, please just commit a fix for the changed reference.

**Comment by ryantm** [comment]
Okay, I fixed eval, but I'm not sure I fixed it correctly. @jtojnar

File: pkgs/development/haskell-modules/hackage-packages.nix [hunk 1]
```diff
@@ -87021,7 +87021,7 @@ self: {
        doHaddock = false;
        description = "GtkSource bindings";
        license = stdenv.lib.licenses.lgpl21;
-     }) {gtksourceview3 = pkgs.gnome3.gtksourceview;};
+     }) {gtksourceview3 = pkgs.gtksourceview;};
```
Comment (jtojnar) [inline pkgs/development/haskell-modules/hackage-packages.nix]: This should be
```suggestion
     }) {gtksourceview3 = pkgs.gtksourceview3;};
```

File: pkgs/development/haskell-modules/hackage-packages.nix [hunk 2]
```diff
@@ -94966,7 +94966,7 @@ self: {
        description = "Binding to the GtkSourceView library";
        license = stdenv.lib.licenses.lgpl21;
        hydraPlatforms = stdenv.lib.platforms.none;
-     }) {gtksourceview3 = pkgs.gnome3.gtksourceview;};
+     }) {gtksourceview3 = pkgs.gtksourceview;};
```
Comment (jtojnar) [inline pkgs/development/haskell-modules/hackage-packages.nix]:
```suggestion
     }) {gtksourceview3 = pkgs.gtksourceview3;};
```

**Comment by jtojnar** [comment]
Otherwise it looks fine.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**